src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -1327,6 +1327,18 @@
     text: >
         az acr connected-registry install renew-credentials -r mycloudregistry -n myconnectedregistry
 """
+
+helps['acr connected-registry repo'] = """
+type: command
+short-summary: Updates all the necessary connected registry sync scope maps repository permissions.
+examples:
+  - name: Adds the 'myconnectedregistry' mode repository permissions to it and its ancestors' sync scope map actions.
```

Comment:
what is `mode repository permissions`?

Reply:
It's my way of saying that if your target connected registry 'myconnectedregistry' is MIRROR it will add the MIRROR permissions to all ancestors. If it is REGISTRY it will add the registry permissions instead.

Reply:
You might want to expand it a little bit. Just make sure the audience has enough context.

Reply:
It might be a bit long now but let me know what you think.

Reply:
Question here - my understanding is that we don't support nested REGISTRY mode; if REGISTRY can only be on top level how should customers understand the behavior for this command?